samples/core/src/test/java/io/oasp/gastronomy/restaurant/general/service/impl/rest/CsvProviderTest.java

```diff
@@ -0,0 +1,21 @@
+package io.oasp.gastronomy.restaurant.general.service.impl.rest;
+
+import static org.junit.Assert.fail;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+/**
+ * @author MLAVIGNE
+ *
+ */
+public class CsvProviderTest {
```

Comment:
This is fine if you are considering it as a TODO where you are working on. In such case please comment your PR and let us know when the time has come to review the PR. 
Otherwise if PR is intended for merge then do not add such code at all.

BTW: please do not add `@author` headers. We already have spent quite some effort to get rid of them. The latest oasp4j-ide config should not contain this in the templates anymore.

Reply:
Indeed this file should have been deleted